backend/app/routers/auth.py

```
@@ -4,16 +1,40 @@
+from app.utils import (
+    auth_utils,
+    create_access_token,
+    verify_token,
+    create_email_verification_token,
+    send_verification_email,
+    verify_reset_token,
+    create_reset_token,
+    send_reset_email,
+)
+from app.database import get_db
 from starlette.responses import JSONResponse
```

Ruff (0.8.2)

Redefinition of unused `JSONResponse` from line 11

(F811)